__opt__stratux__bin__stratux-pre-start.sh

```diff
@@ -10,30 +10,71 @@ function wLog () {
 }
 wLog "Running Stratux Updater Script."
 
+TEMP_DIRECTORY="/boot/firmware/StratuxUpdates"
+
+######################
+# script based update (legacy)
 SCRIPT_MASK="update*stratux*v*.sh"
-TEMP_LOCATION="/boot/firmware/StratuxUpdates/$SCRIPT_MASK"
-UPDATE_LOCATION="/root/$SCRIPT_MASK"
+TEMP_SCRIPT_LOCATION="$TEMP_DIRECTORY/$SCRIPT_MASK"
+SCRIPT_UPDATE_LOCATION="/root/$SCRIPT_MASK"
+
+######################
+# package based update
+PACKAGE_MASK="stratux*.deb"
+# packages are placed here after download, before movement to PACKAGE_UPDATE_LOCATION
+TEMP_PACKAGE_LOCATION="$TEMP_DIRECTORY/$PACKAGE_MASK"
+# packages are placed here for update
+PACKAGE_UPDATE_LOCATION="/root/$PACKAGE_MASK"
 
-if [ -e ${TEMP_LOCATION} ]; then
-	wLog "Found Update Script in $TEMP_LOCATION$SCRIPT_MASK"
-	TEMP_SCRIPT=`ls -1t ${TEMP_LOCATION} | head -1`
-	wLog "Moving Script $TEMP_SCRIPT"
-	cp -r ${TEMP_SCRIPT} /root/
+###############
+# script based update in download location (legacy)
```

Comment:
Can remove all that, too, then.

Reply:
I removed the mention of legacy as we are retaining script based update support.